eloop.c

```diff
@@ -287,9 +287,9 @@ static void eloop_process_pending_signals(void)
 	eloop.signaled = 0;
 
 	if (eloop.pending_terminate) {
-#ifndef CONFIG_NATIVE_WINDOWS
+#if !defined(CONFIG_NATIVE_WINDOWS) && !defined(CONFIG_ZEPHYR)
```

Comment:
If this commit is fixing another `nrf noup` please follow the procedure for that.

Reply:
@krish2718 No chaitanya, it won't fix any commit.

Reply:
whichever commit we start seeing compilation failure, this fix should be part of it.

Reply:
InitIally I did that, u commented to split.

Reply:
Sorry, that looked like an unrelated fix, the commit log didn't explain, hence the confusion.

Reply:
so, it is fixing a `nrf noup`, so, you can keep this commit but follow the squashing policy.

Reply:
Your latest changes have overwritten mine, this and previous commit are missing the `fixup! [nrf noup]` line.